pylint/checkers/strings.py

```diff
@@ -541,15 +562,21 @@ class StringConstantChecker(BaseTokenChecker):
     # Unicode strings.
     UNICODE_ESCAPE_CHARACTERS = 'uUN'
 
+    string_tokens = {}
```

Comment:
this really should be instance attribute, no class attribute.
We also need to be sure that data gets thrown away before switching to new module for analysis (simple reinitialization of dict on the beginning of process tokens should work fine).

Reply:
I tested it manually, it is thrown away.
I'm going to try to make this an instance attribute, but it's bit annoying that you precisely recommended a class attribute in one of your earlier comment :(

Do you want to add a unit test ?
If so, could you please then point me to an existing test I could use as an example to perform such "reinitialization" ?